packages/glab-cli/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://glab-cli.io
+TERMUX_PKG_DESCRIPTION="A GitLab CLI tool bringing GitLab to your command line"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.22.0"
+TERMUX_PKG_SRCURL=https://github.com/profclems/glab/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=4b700d46cf9ee8fe6268e7654326053f4366aa3e072b5c3f3d243930a6e89edc
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="GLAB_VERSION=${TERMUX_PKG_VERSION}"
+
+termux_step_pre_configure() {
+	      termux_setup_golang
```

Comment:
Indent with single tab or alternatively it's better to install [shfmt](https://github.com/mvdan/sh) formatter to do it for you.

You may install it using:
```
go install mvdan.cc/sh/v3/cmd/shfmt@latest
```